packages/devtools_app/lib/src/screens/memory/panes/control/chart_button.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../../../shared/common_widgets.dart';
+import '../../../../shared/globals.dart';
+import '../../primitives/ui.dart';
+
+class ChartButton extends StatelessWidget {
```

Comment:
please merge with master and remove this file - `ChartVisibilityButton` from common_widget.dart should now be used (master has this already)